Dockerfile

```diff
@@ -7,6 +7,14 @@ ENV TOMCAT_HOME /usr/local/tomee
 ENV TOMCAT_CONNECTOR_PROXY_PORT 443
 ENV TOMCAT_CONNECTOR_SCHEME https
 
+# It is assumed that if you override this, you provide suitable values for
+# these system properties. They are used in server.xml to configure the HTTP
+# Connector
+ENV CATALINA_OPTS="\
```

Comment:
This solution introduced some issues for the CI. 
In getting CCLF CI pipeline (which I determined to be more of a priority) this was not adopted in CCLF.